node/src/main/scala/coop/rchain/node/storage/LmdbKeyValueStore.scala

```diff
@@ -0,0 +1,87 @@
+package coop.rchain.node.storage
+
+import java.nio.ByteBuffer
+
+import cats.effect.Sync
+import cats.syntax.all._
+import coop.rchain.shared.Resources.withResource
+import coop.rchain.store.KeyValueStore
+import org.lmdbjava._
+
+final case class Status(
```

Comment:
Could you add a comment explaining what is Status?

Reply:
I left this by mistake. It's not used for now, but the idea was to expose some of storage info through Admin API.